ui-tests/src/styles/theme.scss

```diff
@@ -43,14 +109,14 @@
     --mynah-shadow-button: 0 5px 10px -10px rgba(0, 0, 0, 0.25);
     --mynah-shadow-card: 0 10px 20px -15px rgba(0, 0, 0, 0.25);
     --mynah-shadow-overlay: 0 7px 27px -12px rgba(0, 0, 0, 0.35);
-    --mynah-syntax-code-font-size: 1rem;
-    --mynah-syntax-code-line-height: 1.175rem;
-    --mynah-font-size-xxsmall: 0.75rem;
-    --mynah-font-size-xsmall: 0.85rem;
-    --mynah-font-size-small: 0.95rem;
-    --mynah-font-size-medium: 1rem;
-    --mynah-font-size-large: 1.125rem;
-    --mynah-line-height: 1.175rem;
+    --mynah-syntax-code-font-size: 13px;
```

Comment:
Was wondering this as well ^

Reply:
Very nice question and good catch. The reason we're pushing it to fixed px sizes is to avoid OS based font rendering related **micro** pixel shifts. Even though you see everything exactly the same two screenshots, screenshot tool will not match them for different OSs. Screenshot tester catches some ghost pixels. Instead of letting the OS calculate the px, we're giving pixels. But it doesn't change the fact that we'll still use rem/em in necessary blocks.